src/main/java/http/server/Response.java

```diff
@@ -0,0 +1,51 @@
+package http.server;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Response {
```

Comment:
I considered breaking out some of the functionality of the Response and Request classes into, for example `Request Builder` and `ResponseParser` but ultimately decided that they weren't yet doing enough to justify this.